frontend/view-edit-client/client-home/view-edit-intake-info.component.tsx

```diff
@@ -132,12 +132,14 @@ export default function ViewEditIntakeInfo(props: ViewEditIntakeInfoProps) {
               </tr>
             </tbody>
           </table>
-          <button
-            className="secondary edit-button"
-            onClick={() => setEditing(true)}
-          >
-            Edit
-          </button>
+          {props.editable && (
+            <button
+              className="secondary edit-button"
+              onClick={() => setEditing(true)}
```

Comment:
Should this be setIsEditing?

Reply:
No, see https://github.com/JustUtahCoders/comunidades-unidas-internal/pull/255/files#diff-ba8d1cee557b5029b3796f191da2e5a3R17